tests/store/account.spec.ts

```diff
@@ -81,3 +81,6 @@ describe("Account getters", () => {
     });
   });
 });
+function beforeEach(arg0: () => void) {
```

Comment:
Is this function ever used?

Mocha `beforeEach` hook is usually specified  inside the `describe`

Reply:
not even sure we use this tests, I think they are from the older versions and actually almost useless

Reply:
Maybe remove it then?